ui/app/components/variable-paths.hbs

```diff
@@ -13,7 +13,12 @@
   <tbody>
     {{#each this.folders as |folder|}}
       <tr data-test-folder-row {{on "click" (fn this.handleFolderClick folder.data.absolutePath)}}>
-        <td colspan="3">
+        <td colspan="3"
+          {{keyboard-shortcut 
+            enumerated=true
+            action=(fn this.handleFolderClick folder.data.absolutePath)
```

Comment:
`blocking`: This should accept a namespace.

Reply:
clicking into a folder doesn't do any variable `read`, only variable `list` (like clicking into a folder in a file directory). As such, if the folder is present at all it means the user has access to list at least one variable within it and it should remain clickable. Thus, no namespace passing.

In addition: the concept of "folders" are generated in the ui and do not themselves even *have* namespaces. They can contain many variables from many different namespaces. So nothing to pass in even if it was a good idea to do so.